### ISSUE TITLE
`Search for croaks withing` miles instead of km

### ISSUE THREAD

**Issue by vtomole** [grothedev/fp_flutter#2]
Right now the setting is in kilometers, but since first time users will be in the US, it will be more user friendly for it to be in miles.

**Comment by grothedev** [comment]
yeah you're probably right, but i kind of want to encourage people to use sensible units instead of the US system.

**Comment by vtomole** [comment]
Well, i encourage you to survey more of your users to see what they would prefer.
